lib/percentmanager.go

```diff
@@ -45,7 +90,31 @@ func NewProgressWriter() progress.Writer {
 	pw.SetTrackerPosition(progress.PositionRight)
 	pw.SetUpdateFrequency(time.Millisecond * 100)
 	pw.Style().Options.PercentFormat = "%4.1f%%"
-	pw.Style().Colors = CuteColors
+
+	colorsSet := CuteColors
+
+	var targetUser int
+	baseUser, exists := os.LookupEnv("SUDO_UID")
+	if !exists || baseUser == "" {
+		targetUser = 0
+	} else {
+		var err error
+		targetUser, err = strconv.Atoi(baseUser)
+		if err != nil {
+			slog.Error("Failed parsing provided user as UID", slog.String("user_value", baseUser))
+			return pw
+		}
+	}
+
+	if targetUser != 0 {
+		cli := []string{"gsettings", "get", "org.gnome.desktop.interface", "accent-color"}
```

Comment:
calling gsettings directly is probably not what we want. There's a dbus interface we can use instead: https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Settings.html

Reply:
OMG WHAT I DIDNT KNOW